src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -825,6 +825,36 @@ def generateBaseMesh(region, options):
                 bodyMeshGroup.addElement(element)
                 elementIdentifier += 1
 
+        # create 2d mesh groups
+        fm.defineAllFaces()
+        bodyGroup.addSubelements()
```

Comment:
Change these lines to the following to pick up all annotation groups you have now and any you add later:
```
for annotationGroup in annotationGroups:
    annotationGroup.addSubelements()
```